samples/java-customer-registry-quickstart/src/main/proto/customer/api/customer_api.proto

```diff
@@ -12,7 +12,6 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// tag::declarations[]
```

Comment:
I was sure because I run the documentation locally and it didn't produce any error, but indeed, if that's referenced from the main doc site, then I need to add it back.

However, we should limit the amount of things in this repo that is not verified by this build. Those pages should be here instead. 

Anyway, I will check it out and most probably revert it.